File: csci3130projectJavaFiles/src/java/genericgame1067/Character.java
package genericgame1067;

public class Character {

    //attributes
    private int charaId; //character id
    private int accountId; //account id
    private String charaName; //character's name
    private int dailyPoint; //character's daily point
    private String picture; //character's picture(stored file name)

    //empty constructor
    public Character()
    {
    }

    //constructor w/ parameter
    public Character(int charaId, int accountId, String charaName, 
        int dailyPoint, String picture)
    {
    	this.charaId=charaId;
    	this.accountId=accountId;
    	this.charaName=charaName;
    	this.dailyPoint=dailyPoint;
	this.picture=picture;
    }

    public int getAccountID(){
	return accountId;
    }
	
    public int getCharaId() {
	return charaId;
    }
    public void setCharaId(int charaId) {
	this.charaId = charaId;
    }
    public void setAccountId(int accountId) {
	this.accountId=accountId;
    }
    public String getCharaName() {
	return charaName;
    }

    public void setCharaName(String charaName) {
	this.charaName = charaName;
    }

    public int getDailyPoint() {
	return dailyPoint;
    }

    public void setDailyPoint(int dailyPoint) {
	this.dailyPoint = dailyPoint;
    }

    public String getPicture() {
	return picture;
    }

    public void setPicture(String picture) {
    	this.picture = picture;
    }
    @Override
    public String toString() {
    	return "Character [charaId=" + charaId + ", accountId=" + accountId
        	+ ", charaName=" + charaName + ", dailyPoint=" + dailyPoint
		+ ", picture=" + picture + "]";
    }

}

File: phpfiles/Character.php
<?php


class Character extends user {

    /** attributes */
    public $charaId;
    public $accountId;
    public $charaName;
    public $dailyPoints;
    public $picture; // character's picture (stored file name)

    /** constructor */
    function __construct($charaId, $accountId, $charaName, $dailyPoints, $picture){
        $this->charaId = $charaId;
        $this->accountId = $accountId;
        $this->charaName = $charaName;
        $this->dailyPoints = $dailyPoints;
        $this->picture = $picture;
    }

    /** set methods */
    function setCharaId($charaId){
        $this->charaId = $charaId;
    }

    function setAccountId($accountId){
        $this->accountId = $accountId;
    }

    function setCharaName($charaName){
        $this->charaName = $charaName;
    }

    function setDailyPoints($dailyPoints){
        $this->dailyPoints = $dailyPoints;
    }

    function setPicture($picture){
        $this->picture = $picture;
    }

    /** get methods */
    function getCharaId(){
        echo $this->charaId;
    }

    function getAccountId(){
        echo $this->accountId;
    }

    function getCharaName(){
        echo $this->charaName;
    }

    function getDailyPoints(){
        echo $this->dailyPoints;
    }

     function getPicture(){
        return $this->picture;
    }

    function __toString() {
        return 'Character [' . charaId . "Account ID: " . accountId . "Character Name: "
            . charaName . "Points Today: " . dailyPoints . + "Picture: " . picture . "]";
    }


} //end of class